ecommerce/products/base.py

```diff
@@ -0,0 +1,82 @@
+from django.db import models
+
+
+class Rarity:
+    COMMON = 'Common'
+    UNCOMMON = 'Uncommon'
+    RARE = 'Rare'
+    ULTRA_RARE = 'Ultra Rare'
+    AMAZING_RARE = 'Amazing Rare'
+    SECRET_RARE = 'Secret Rare'
+    CHOICES = [
+        (COMMON, 'Common'),
+        (UNCOMMON, 'Uncommon'),
+        (RARE, 'Rare'),
+        (ULTRA_RARE, 'Ultra Rare'),
+        (AMAZING_RARE, 'Amazing Rare'),
+        (SECRET_RARE, 'Secret Rare')
+    ]
+
+class Languages:
+    ENGLISH = 'English'
+    JAPANESE = 'Japanese'
+    FRENCH = 'French'
+    SPANISH = 'Spanish'
+    GERMAN = 'German'
+    CHINESE ='Chinese'
+    KOREAN = 'Korean'
+    CHOICES = [(ENGLISH, 'English'),
+               (JAPANESE, 'Japanese'),
+               (FRENCH, 'French'),
+               (SPANISH, 'Spanish'),
+               (GERMAN, 'German'),
+               (CHINESE, 'Chinese'),
+               (KOREAN, 'Korean')
+               ]
```

Comment:
Formattng on this is a tiny bi t incosistent with how you did it on `Rarity`. It's better the way you did it on rarity so maybe update it to be consistent throughout the whole script